.github/workflows/ci.yml

```diff
@@ -45,22 +45,27 @@ jobs:
           mkdir -p $NXF_SINGULARITY_CACHEDIR
           mkdir -p $NXF_SINGULARITY_LIBRARYDIR
 
-      - name: Download test data
+      - name: Download Tiny (RAPID) test data
```

Comment:
It seems this test data being used by rapid test and non-rapid test, why you add a RAPID word here?

Reply:
This was whilst I was putting together a bigger test data set which we no longer need. I'll remove.